pkg/frontend/adminactions/kubeactions.go

```diff
@@ -27,6 +29,7 @@ type KubeActions interface {
 	KubeCreateOrUpdate(ctx context.Context, obj *unstructured.Unstructured) error
 	KubeDelete(ctx context.Context, groupKind, namespace, name string) error
 	Upgrade(ctx context.Context, upgradeY bool) error
+	KubeGetPodLogs(ctx context.Context, namespace string, name string, containerName string) ([]byte, error)
```

Comment:
Just a nit for consistency with other methods. If you have multiple params of the same type you can do this:

```suggestion
	KubeGetPodLogs(ctx context.Context, namespace, name, containerName string) ([]byte, error)
```

Reply:
done